crates/nu-command/src/core_commands/help.rs

```diff
@@ -96,14 +98,22 @@ fn help(
             let mut vals = vec![];
 
             let decl = engine_state.get_decl(decl_id);
-            let sig = decl.signature();
+            let sig = decl.signature().update_from_command(decl.borrow());
+
+            let key = sig.name;
+            let usage = sig.usage;
+            let search_terms = sig.search_terms;
+            let matches_term = if search_terms.is_empty() {
```

Comment:
@kubouch Is this a typo or am I missing something?

I think this will only look at the search terms if the search terms are empty.

Reply:
I'm 99% sure this is a typo, will submit a PR.

Reply:
@rgwood  Yeah, a typo, please feel free to submit a PR, I'm on holiday now and don't have much time to hack on Nushell.